### ISSUE TITLE
Multiple code blocks with same language not working - code panes not switching

### ISSUE THREAD

**Issue by npabion** [yacir/markdown-fenced-code-tabs#12]
Using something like:

\`\`\`java fct_label="One"
pubic static void oneTest();
\`\`\`
\`\`\`java fct_label="Two"
public static void twoTest();
\`\`\`

Generates the 2 tabs but the links are the same, e.g. \<a href="#tab-22-java"

Sample generated from the above:
```html
<div class="code-nav-container">    
    <ul class="nav nav-tabs">        
        <li class="nav-item active">
            <a href="#tab-22-java" aria-controls="tab-22-java" role="tab" data-toggle="tab" data-lang="java">One</a>
        </li>        
        <li class="nav-item">
            <a href="#tab-22-java" aria-controls="tab-22-java" role="tab" data-toggle="tab" data-lang="java">Two</a>
        </li>
        
    </ul>
    
    <div class="tab-content">        
        <div role="tabpanel" class="tab-pane active" id="tab-22-java">
            <div class="codehilite" id="__code_22"><button class="md-clipboard" title="Copy to clipboard" data-clipboard-target="#__code_22 pre, #__code_22 code"><span class="md-clipboard__message"></span></button><pre><span></span><span class="n">pubic</span> <span class="kd">static</span> <span class="kt">void</span> <span class="nf">oneTest</span><span class="o">();</span>
            </pre></div>
        </div>    
        <div role="tabpanel" class="tab-pane" id="tab-22-java">
            <div class="codehilite" id="__code_23"><button class="md-clipboard" title="Copy to clipboard" data-clipboard-target="#__code_23 pre, #__code_23 code"><span class="md-clipboard__message"></span></button><pre><span></span><span class="kd">public</span> <span class="kd">static</span> <span class="kt">void</span> <span class="nf">twoTest</span><span class="o">();</span>
            </pre></div>
        </div>
    </div>
</div>
````

**Comment by otisg** [comment]
Is the underlying issue (and solution) the same as for #11 and #10 @yacir ?

**Comment by yacir** [comment]
@otisg  @npabion sorry was busy last days :/.

I've fixed it in the [1.0.0](https://github.com/yacir/markdown-fenced-code-tabs/tree/1.0.0/) which, i think, will be released today :)

**Comment by yacir** [comment]
@npabion @otisg it's finally done. Please check the [new docs](https://yacir.github.io/markdown-fenced-code-tabs/).
